src/app/_utils/convertISOTime.ts

```diff
@@ -0,0 +1,25 @@
+type extractTimeFromISOType = {
+  day: number
+  month: string
+  time: string
+}
```

Comment:
When return types are used only once, like here, I believe they are unnecessary. We can let TS infer the return type and simplify the code.

```suggestion
```